jenkins/run-unittests

```diff
@@ -24,6 +24,13 @@ if [[ ${flake_sts} -ne 0 ]]; then
 fi
 printf -- "\n"
 
+pytest {posargs} ./server/lib/pbench/test/unit
```

Comment:
For the Jenkins tests, we don't need `{posargs}`, and its present does not resolve because the syntax is for `tox` not `bash`.

Reply:
I looked on Jenkins and saw that it is failing because it cannot find any file for running pytests `ERROR: file not found: ./server/lib/pbench/test/unit` which will not be available until pbench-put-shim is merged.

Reply:
> I looked on Jenkins and saw that it is failing because it cannot find any file for running pytests `ERROR: file not found: ./server/lib/pbench/test/unit` which will not be available until pbench-put-shim is merged.

In that case, you might want to set this PR's upstream branch to your put-shim PR; that makes the dependency obvious, and will also allow it to actually run now.

Or, given that it's now a small block in a single file, maybe integrating pytest runs with Jenkins should just be part of the PR that adds the pytest code? It doesn't necessarily make sense to break *every* change into a separate PR. 😁